<launch>
    <node name="turtlesim_node" pkg="turtlesim" type="turtlesim_node" />

    <!-- setting up the workstations  -->
    <param name="/num_rows_workstations" type="double" value="1.0" />
    <param name="/num_workstations" type="double" value="2.0" />    
    <param name="/num_buffers" type="double" value="2.0" />
    
    <param name="/infeed_origin_x" type="double" value="1.0" />
    <param name="/infeed_origin_y" type="double" value="1.0" />
    <param name="/outfeed_origin_x" type="double" value="10.0" />
    <param name="/outfeed_origin_y" type="double" value="10.0" />


    <!-- setting up the turtlebots
        the first one is always "turtle1"-->
    <param name="/turtle_origin_xy" type="double" value="0.5" />
    <param name="/num_turtlebots" type="double" value="11.0" />
    <param name="/turtle_names" type="string" value="TB" />

    <node name="turtle_build_factory" pkg="turtle_factory" type="turtle_build_factory" />

    <!-- setting up the communication nodes-->
    <node name="web_comm_server_node" pkg="agm_comm" type="web_comm_server_node" />
    <node name="turtle_agm_worker_node" pkg="turtle_factory" type="turtle_agm_worker_node" args="668f1a4fe9bcef706b10fc29 move_base TB01" output="screen"/>
    <!---->
    <node name="turtle_agm_worker_node2" pkg="turtle_factory" type="turtle_agm_worker_node" args="6695852ceb3f587734a830be move_base TB02" output="screen"/>
    <node name="turtle_agm_worker_node3" pkg="turtle_factory" type="turtle_agm_worker_node" args="66970905647a83c0be34859a move_base TB03" output="screen"/>
</launch>